src/main/java/com/github/zfl4wless/ui/pages/MenuPage.java

```diff
@@ -0,0 +1,24 @@
+package com.github.zfl4wless.ui.pages;
+
+import com.github.zfl4wless.Main;
+import com.github.zfl4wless.ui.pages.listeners.QuitButtonActionListener;
+import com.github.zfl4wless.ui.pages.listeners.StartButtonActionListener;
+
+import javax.swing.*;
+
+public class MenuPage {
+    private static final JPanel page = Main.getApp().getPageHandler().createPage();
+
+    public static JPanel view() {
```

Comment:
```suggestion
    public static JPanel render() {
```
would suggest a refactor to better describe the method, idk